.smalltalk.ston

```diff
@@ -6,11 +6,5 @@ SmalltalkCISpec {
       #load : 'Tests',
       #platforms : [ #squeak ]
     }
-  ],
-  #testing : {
-    #coverage : {
-      #packages : [ 'ChangesReloaded-Core.*' ]
-    },
-    #packages : [ 'ChangesReloaded-Tests.*' ]
-  }
```

Comment:
This change will break our coverage reporting. Was this intentional?

Reply:
nope